workflow/rules/common.smk

```diff
@@ -173,6 +173,7 @@ def all_input(wildcards):
             [
                 "results/plots/mean-var/{model}.mean-variance-plot.pdf",
                 "results/plots/volcano/{model}.volcano-plots.pdf",
```

Comment:
I think we should remove the old volcano plot to avoid confusion. One thing to double-check, is that plotly enables an export of a figure into something like a PDF or JPEG.

Reply:
plotly + kaleido can generate those (and you can always hit the save button in the top right corner of html plots to get a png render of that plot; you just don't have much control about the png's resulting size in that case).
Do we replace those pdfs with plotly versions or do we remove them altogether?